ruby-challenges.rb

```diff
@@ -30,3 +63,32 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+class Bike 
+  attr_accessor :model, :wheels, :current_speed, :pedal_faster,
```

Comment:
Since you have a method defined for pedal_faster, there is no need for a getter and setter for it in attr_accessor.